### DISCUSSION TITLE
Older Nvidia Graphics Cards

### DISCUSSION THREAD

**Discussion by Gorilli09** [86Box/86Box#3879] [Ideas]
Can you emulate older Nvidia graphics cards. The official Nvidia website still has the drivers.

**Comment by TC1995** [comment]
Find a proper documentation of the nvidia registers/ports/acceleration details.

**Comment by sskras** [comment]
> Can you emulate older Nvidia graphics cards.

Which code names are you interested in, please?
https://nouveau.freedesktop.org/CodeNames.html

**Reply by Gorilli09** [reply]
Nvidia NVS

**Reply by sskras** [reply]
Ah, so it's NV17-NV18 chips.

**Comment by eddmanx** [comment]
This comes up constantly. For it to be possible to be done in any practical sense, detailed, register-level hardware documentation (for both 2D and 3D) is needed. Last I checked, there isn't any, and they're also confidential. Maybe ask nvidia if they're willing to publish them.

An alternative would be reverse engineering, but that's a rather difficult and very time consuming task. Unless some people show up who are willing to take on such a challenge, don't expect it to happen any time soon.

Besides, even if there were documents, you won't get emulation for anything higher than a TNT/2.
